IPython/utils/PyColorize.py

```diff
@@ -59,6 +59,21 @@
 else:
     from StringIO import StringIO
 
+import pygments.styles
+
+
+from traitlets.config import Configurable
+from traitlets import Unicode, Bool
+
+
+available_themes = lambda : [s for s in pygments.styles.get_all_styles()]+['NoColor','LightBG','Linux']
+
+class Colorable(Configurable):
```

Comment:
> Let's put this in a new module, rather than reusing PyColorize - PyColorize is specifically about colouring Python source code, whereas this is general machinery for colour output.

Do you really want a module just for 15 lines of code ?